app/lib/steps/helper.rb

```diff
@@ -49,6 +49,7 @@ def all_sections
          ApplicantDetailsSection,
          IncomeSection,
          PartnerSection,
+         PostGrossIncomeEarlyEligibilitySection,
```

Comment:
I'm guessing we'll also have a `PostCapitalEarlyEligibilitySection` section/screen (potential not `PostDisposableIncomeEarlyEligibilitySection` as we need to know it for contributions)?

Reply:
I think it's kinda the other way round. There's no point doing a `PostCapitalEarlyEligibilitySection` because once you've completed the capital questions you're at the end of the flow anyway. 

With disposable income, the point is that we shouldn't try to obtain a result part-way through the outgoings questions, but once we've reached the _end_ of the outgoins questions, a `PostDisposableIncomeEarlyEligibilitySection` would allow us to potentially skip the capital questions entirely if we know a client is ineligible.

(The complication is that _at the moment_ the last disposable income screen is almost at the end of the flow, so there's very little time saving here either, but that might change if we think a reshuffle could save some people some time)

Reply:
I see, that makes more sense.